release/pyproject.toml

```diff
@@ -10,11 +10,7 @@ classifiers = [
 dependencies = [
     "gitpython==3.1.43",
 ]
-optional-dependencies.dev = [
-    "pip==24.2",
-    "pip-tools==7.4.1", # To add hashes to requirements
-    "pipx==1.7.1",
-]
+optional-dependencies.dev = [  ]
```

Comment:
Why not omit?

Reply:
I had removed it and `pip compile` complained so I put it back in. But upon looking better (thanks) a better solution is to remove the `--extra dev` flag. Oops.